src/routes/identity-manager/[id].svelte

```diff
@@ -0,0 +1,133 @@
+<script context="module" lang="ts">
+    export const prerender = true
+</script>
+
+<script lang="ts">
+    import { CreateCredentialModal, DEFAULT_VCS_TEMPLATES, IdentityDetails } from '$lib'
+    import { authenticatedUserRole } from '$lib/app/base'
+    import {
+        getIdentitiy,
+        getVerifiableCredentials,
+        searchIdentityByDID,
+        selectedIdentity,
+        updateIdentityInSearchResults,
+        loading,
+        getIdentityClaim,
+    } from '$lib/app/identity'
+    import { UserRoles, type VerifiableCredentialTemplate } from '$lib/app/types'
+    import type { ActionButton } from '$lib/app/types'
+    import { Col, Container, Row } from 'sveltestrap'
+    import { Icon } from '$lib/components'
+    import type { User } from '@iota/is-client'
+    import { get } from 'svelte/store'
+    import { page } from '$app/stores'
+    import { onDestroy, onMount } from 'svelte'
+    import { goto } from '$app/navigation'
+
+    let credentialsTemplate: VerifiableCredentialTemplate[] = DEFAULT_VCS_TEMPLATES
+    let isCreateCredentialModalOpen = false
+
+    let identity: User = undefined
+    let userRole: UserRoles = undefined
+    let isLoading: boolean = false
+    const unsubscribeIdentity = selectedIdentity.subscribe((id) => (identity = id))
```

Comment:
I think with auto-subscriptions this boiler plate subscription pattern is unnecessary. Have a look here how to use reactivity on stores: https://svelte.dev/tutorial/auto-subscriptions